src/pages/EditMonitor.vue

```diff
@@ -246,19 +249,66 @@
                             </div>
 
+                            <!-- SNMP Monitor Type -->
+                            <div v-if="monitor.type === 'snmp'" class="my-3">
+                                <label for="snmp_community_string" class="form-label">{{ $t("Community String") }}</label>
+                                <input id="snmp_community_string" v-model="monitor.snmpCommunityString" type="text" class="form-control" required placeholder="Example: public">
+                            </div>
+
+                            <div v-if="monitor.type === 'snmp'" class="my-3">
+                                <label for="snmp_oid" class="form-label">{{ $t("OID (Object Identifier)") }}</label>
+                                <input id="snmp_oid" v-model="monitor.snmpOid" type="text" class="form-control" required pattern="^([0-2])((\.0)|(\.[1-9][0-9]*))*$" title="Please enter a valid OID, e.g. 1.3.6.1.4.1.9.6.1.101" placeholder="Example: 1.3.6.1.4.1.9.6.1.101">
+                            </div>
+
+                            <div v-if="monitor.type === 'snmp'" class="my-3">
+                                <div class="d-flex align-items-start">
+                                    <div class="me-2">
+                                        <label for="snmp_condition" class="form-label">{{ $t("Condition") }}</label>
+                                        <select id="snmp_condition" v-model="monitor.snmpCondition" class="form-select me-3" required>
+                                            <option value=">">&gt;</option>
+                                            <option value=">=">&gt;=</option>
+                                            <option value="<">&lt;</option>
+                                            <option value="<=">&lt;=</option>
+                                            <option value="==">==</option>
+                                            <option value="contains">contains</option>
+                                        </select>
+                                    </div>
+                                    <div class="flex-grow-1">
+                                        <label for="snmp_control_value" class="form-label">{{ $t("Control Value") }}</label>
+                                        <input v-if="monitor.snmpCondition !== 'contains' && monitor.snmpCondition !== '=='" id="snmp_control_value" v-model="monitor.snmpControlValue" type="number" class="form-control" required step=".01">
+                                        <input v-else id="snmp_control_value" v-model="monitor.snmpControlValue" type="text" class="form-control" required>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div v-if="monitor.type === 'snmp'" class="my-3">
+                                <label for="snmp_version" class="form-label">{{ $t("SNMP Version") }}</label>
```

Comment:
Is snmp backwards compatible?
what is the difference between these versions?

=> is a short helptext (1 sentence each) how these selections differ and how to choose one option possible/advantageous?

Reply:
SNMP versions are generally backward compatible, but newer versions offer improved features and security enhancements. SNMPv1 is the original version with limited security and functionality. SNMPv2 added features like bulk data transfer and improved error handling. SNMPv3 introduced robust security features including encryption and authentication.

However I'm not using the security features of SNMPv3 so perhaps I remove that option and implement later...

I will add the helptext(s) and remove SNMPv3 as an option.

Reply:
> remove SNMPv3 as an option

Resolved with 86b997c664edb8cbeec796b8e04ecd24b5ae753f

>I will add the helptext(s)

Resolved with 4ef66b37601c398870c32810ffadf3365c72a70f. Here's a screenshot:
![image](https://github.com/louislam/uptime-kuma/assets/9312603/3e2fc499-c824-444d-b570-b84ae322aaf8)